www/scipylib/building/windows.rst

```diff
@@ -32,6 +32,109 @@ information than this document will provide.
 .. _OpenBLAS: https://github.com/xianyi/OpenBLAS
 .. _LAPACK: http://www.netlib.org/lapack/
 
+
+Building the Released SciPy
+---------------------------
+
+This section provides the step-by-step process to build the released scipy. If you want
+to build completely from source, you should estimate at least three hours to build all
+libraries and compile SciPy. Feel free to stop and inspect any step at any time, but
+we're mostly just going to breeze through this. More in-depth documentation is provided 
+below.
+
+First, download all of the following software concurrently:
+
+1) MinGW-w64 from https://mingw-w64.org
+2) Microsoft Visual Studio 2015 or 2017 Community Edition (available from Microsoft)
+3) git from https://git-scm.org/
+4) MSYS2 from http://www.msys2.org/
+5) Python **with pip** from https://python.org/
+
+While that's downloading let's outline our plan of action. We're going to:
+
+1) Build OpenBLAS using the MinGW-w64 compiler suite, using the build system from MSYS2
+2) Copy the associated libraries to the `Python\\Lib` directory
+3) Build SciPy with a hybrid MinGW-w64 toolchain
+
+Once git installed, open a terminal in your favorite directory and type the following:
```

Comment:
Grammo - "One git is installed, "

Reply:
I think you mean a git bash terminal here?   I can't remember whether msys git puts git on the Windows PATH by default, but it's certainly an option not to have it on the path.  So `cmd` might not work here.

Can't you install git via msys2?  Wouldn't that be simpler overall (don't need two installers, two terminals)?